deployment_steps_cdk.adoc

```diff
@@ -1 +1,35 @@
-Deployment takes about {deployment_time} to complete.
\ No newline at end of file
+To deploy {partner-product-short-name} on AWS, complete the following steps. The stack takes about {deployment_time} to launch.
+
+. Confirm that you have the https://docs.aws.amazon.com/cdk/v2/guide/getting_started.html#getting_started_prerequisites[prerequisites^] to install the AWS CDK.
+. Install the AWS CDK.
+. Clone the {partner-product-short-name} Quick Start repository:
+```
+git clone https://github.com/aws-quickstart/quickstart-{partner-product-name}.git
+cd quickstart-{partner-product-name}
+```
+[start=4]
+. Build the project:
+```
+npm install
+npm run build
+```
+[start=5]
+. Bootstrap your AWS environment:
+```
+cdk bootstrap
+```
+[start=6]
+. Deploy to a new VPC:
+```
+npm run build && cdk deploy
+```
+[start=7]
+. Alternatively, deploy the stack to an existing VPC:
+```
+npm run build && cdk deploy -c ExistingVPC=true --parameters VpcId=vpc-0c7910af2c48bffff --parameters PrivSubnets=subnet-08a3625e03ebbffff,subnet-0cb32f56d8757ffff
```

Comment:
I think this and the section below will be specific to the QS being developed. So I'm not sure if this is the correct way to approach it by having details listed?